2-networks/modules/standard_shared_vpc/main.tf

```diff
@@ -19,41 +19,39 @@
  *****************************************/
 
 module "main" {
-  source          = "terraform-google-modules/network/google"
-  version         = "~> 2.0"
-  project_id      = var.project_id
-  network_name    = var.network_name
-  shared_vpc_host = "true"
+  source                                 = "terraform-google-modules/network/google"
+  version                                = "~> 2.0"
+  project_id                             = var.project_id
+  network_name                           = var.network_name
+  shared_vpc_host                        = "true"
+  delete_default_internet_gateway_routes = "true"
 
   subnets          = var.subnets
   secondary_ranges = var.secondary_ranges
 
   routes = [
     {
-      name              = "gcp-windows-activation"
-      description       = "Route through IGW to allow Windows kms activation."
+      name              = "egress-internet"
+      description       = "Tag based route through IGW to access internet"
```

Comment:
I think the default pattern should probably be Cloud NAT instead?

Reply:
So this is still using Cloud NAT, its just by default you do not have a path to the internet if that makes sense?

Reply:
For context, the PR changes the default behavior to:
- Access and routes available to Google APIs but no internet access
- egress-internet tag must be applied to reach anything on the internet via Cloud NAT

Reply:
Got it.